src/ainft.ts

```diff
@@ -141,12 +150,25 @@ export default class AinftJs {
     this.setSigner(signer);
   }
 
+  async open() {
```

Comment:
could you add some tsdoc? e.g. 'Opens a session with the blockchain endpoint'

Reply:
done!